js/src/admin/components/AuthPage.js

```diff
@@ -0,0 +1,207 @@
+import Page from './Page';
+import Checkbox from '../../common/components/Checkbox';
+import Button from '../../common/components/Button';
+import Alert from '../../common/components/Alert';
+import Switch from '../../common/components/Switch';
+import LoadingIndicator from '../../common/components/LoadingIndicator';
+import ItemList from '../../common/utils/ItemList';
+import icon from '../../common/helpers/icon';
+import saveSettings from '../utils/saveSettings';
+
+export default class AuthPage extends Page {
+  init() {
+    super.init();
+
+    this.saving = false;
+    this.refresh();
+  }
+
+  refresh() {
+    this.loading = true;
```

Comment:
an `m.redraw()` call is probably needed here, otherwise it doesn't refresh until `this.loading` is set back to `false` at the end of the function.

Does the loading indicator make any sense though ? It's a synchronous operation and I don't see what could be holding the process enough to necessitate a loading indicator.

Reply:
True, fixed by removing the loading indicator.